docs/component/open-attestation.md

```diff
@@ -4,8 +4,303 @@ title: OpenAttestation Wrapper SDK (javascript)
 sidebar_label: Wrapper SDK (javascript)
 ---
 
-Overview
+OA provides a wrapper tool for node platform, to prepare your documents before issuing them into the Blockchain.
```

Comment:
I think it's good that we describe the things one can do with this SDK upfront so they do not need to comb through the entire document to figure it out. 

As example will be:

The OpenAttestation SDK (javascript) is a [npm module](https://www.npmjs.com/package/@govtechsg/open-attestation) that allows you to manipulate OA documents programtically. This is useful if you are building your own API or web components. Some common use cases where you need this module: 

1. Generating OA documents from external data sources programatically
1. Extracting data from OA documents programatically
1. Programtic selective disclosure of data fields from issued OA documents 

This module does not provide the following functionality: 

1. Programatic verification of OA documents (refer to Verification SDK (javascript))
1. Encryption or decryption of OA documents (refer to Encryption SDK (javascript))
1. Programatic issuance/revocation of document on the Ethereum blockchain

Reply:
For the first 3 points, we can hyperlink to the sections below directly as well.